dist/bin/reacli.js

```diff
@@ -205,13 +241,17 @@ var reactCli = function reactCli(args) {
 	var options = {};
 	var useFlow = /\x2D\x2Dflow|\x2Df/.test(args);
 	var useScss = /\x2D\x2Dscss/.test(args);
+	var useRedux = /\x2D\x2Dredux|\x2Dr/.test(args);
 
 	if (useFlow) {
 		options = Object.assign(options, { flow: true });
 	}
 	if (useScss) {
 		options = Object.assign(options, { scss: true });
 	}
+	if (useRedux) {
+		options = Object.assign(options, { redux: true });
```

Comment:
why don't you use spread syntax?
`options = { ...options, redux: true}`
or better yet:
```
const options = {
  flow: useScss,
  scss:  /\x2D\x2Dscss/.test(args),
  ...
```
or just use `commander` it's really cool :)

Reply:
Hi !

I did not use the spread operator yet because of the babel configuration of the project. I did not change it since I know @leoco is working on setting up `commander`. 😊 

So yes, we will be using `commander` starting tomorrow since the related PR should be proposed. 😉

Reply:
By the way, you commented on the `dist/bin/reacli.js` file but it's the one generated by `build` script. Reading `bin/reacli.js` might be easier 😉

Reply:
oh My bad :)  But dist directory should be in .gitignore and package.json should have a postinstall script:
```
"scripts": {
  "build": "babel ....",
  "postinstall": "npm run build",
```

Reply:
Thx for all the feedbacks, I will definitely create issues on both template solution and post install!